python/src/einsum_onnx.py

```diff
@@ -632,11 +711,31 @@ def einsum_decomposed_model_test():
         inputs = [ np.random.rand(*shape) for shape in ishapes ]
         expected = np.einsum(equation, *inputs)
         model = einsum_decomposed_model(equation, ishapes, np.float64)
+        name = equation.replace(',', '_comma_').replace('->', '_arrow_').replace(' ', '').replace('...', '_ellipses_')
+        path = f"output/out%03d_{name}.py" % counter
+        counter += 1
+        print(equation, ishapes, path)
+        try:
+            import onnxconverter_common.onnx2py
+            import os
+            try:
+                os.mkdir("output")
+            except OSError:
+                pass
+            model_trace = onnxconverter_common.onnx2py.convert(model, path)
+            py_obj = onnxconverter_common.onnx2py.TracingObject.get_py_obj(model_trace)
+            print(repr(model_trace))
+            needs_install = False
```

Comment:
how about moving this up next to `counter = 0` in line 659 above? otherwise, in theory, this could overwrite an `need_install = True` assignment from an earlier iteration